textstat/textstat.py

```diff
@@ -88,6 +84,18 @@ def _cache_clear(self):
         for method in caching_methods:
             getattr(self, method).cache_clear()
 
+   def _legacy_round(self, number, points=0):
+        """
+        Function to round floating point outputs for backwards compatibility.
+        Rounding can be turned off by creating an instance of the class
+        textstatistics and setting the attribute round_outputs to False
+        """
+        if self.round_outputs:
+            p = 10 ** points
+            return float(math.floor((number * p) + math.copysign(0.5, number))) / p
+        else:
+            return number
+
```

Comment:
The build is currently failing:
```
E     File "/home/runner/work/textstat/textstat/textstat/textstat.py", line 87
E       def _legacy_round(self, number, points=0):
E                                                ^
E   IndentationError: unindent does not match any outer indentation level
```
It seems you have 3 space indent on the `_legacy_round` method.